### PR TITLE
Client connect

### PR DESCRIPTION
**Pull request by TimSimpsonR**
With this change the tests can now connect to the RDL api.

### CODE REVIEW

File: tests/integration/tests/util/__init__.py [hunk 1]
```diff
@@ -142,9 +143,18 @@ def create_dbaas_client(user):
     """Creates a rich client for the RedDwarf API using the test config."""
     test_config.nova.ensure_started()
     dbaas = Dbaas(user.auth_user, user.auth_key,
-                  user.tenant, test_config.reddwarf_auth_url)
+                  user.tenant, test_config.reddwarf_auth_url,
+                  service_type='reddwarf')
     dbaas.authenticate()
-    return dbaas
+    with Check() as check:
```
Comment (cp16net) [inline tests/integration/tests/util/__init__.py:149]: this is awesome:exclamation: :thumbsup:

**Comment by cp16net** [comment]
+1 :godmode:

File: scripts/conf/test.conf [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "white_box":false,
     "use_local_ovz":false,
     "use_venv":false,
-    "dbaas_url":"http://localhost:8779/v0.1",
+    "dbaas_url":"http://10.0.2.15:8779/v0.1",
```
Comment (cp16net) [inline scripts/conf/test.conf]: Why would the ip address be required and localhost not work? i dont think this would work for my environment currently.

Comment (TimSimpsonR) [inline scripts/conf/test.conf]: You're right. I figured out the correct way to do this and have committed.  :)

File: scripts/redstack [hunk 3]
```diff
@@ -551,9 +598,9 @@ cmd_stop() {
 cmd_int_tests() {
     exclaim "Running Reddwarf Integration Tests..."
     export TEST_CONF=/integration/scripts/conf/test.conf
-    NEW_PATH=$PYTHONPATH:$PATH_REDDWARF:$PATH_REDDWARF_INT_TESTS/integration
-    echo "Python path : $NEW_PATH"
-    export PYTHONPATH=$NEW_PATH
+    NEW_PATH=$PYTHONPATH:$PATH_REDDWARF_INT_TESTS/integration
+    #echo "Python path : $NEW_PATH"
+    #export PYTHONPATH=$NEW_PATH
```
Comment (hub-cap) [inline scripts/redstack]: remove?

File: scripts/redstack [hunk 4]
```diff
@@ -280,12 +312,27 @@ SERVICE_PASSWORD=$SERVICE_PASSWORD" > localrc
...
 
 ensure_local_sources() {
+    exclaim "Making sure local sources are up to date..."
     # this method will only clone if the sources dont exist,
     # or if RECLONE=yes is set
     git_clone $REPO_REDDWARF $PATH_REDDWARF $BRANCH_REDDWARF
     git_clone $REPO_REDDWARFCLIENT $PATH_PYTHON_REDDWARFCLIENT $BRANCH_REDDWARFCLIENT
 }
 
+install_rdl_test_requires() {
```
Comment (hub-cap) [inline scripts/redstack]: is this the same thing that i did earlier? im confused why it says you are adding it... did u just move it?

File: scripts/redstack (hunk 3 above)
Comment (TimSimpsonR) [inline scripts/redstack]: Yep! Done.

File: scripts/redstack (hunk 4 above)
Comment (TimSimpsonR) [inline scripts/redstack]: My bad, TERMINATED.

File: scripts/redstack [hunk 5]
```diff
@@ -211,9 +221,31 @@ install_packages_2() {
...
+    sudo pip install openstack.nose_plugin
+    #sudo pip install proboscis
+
+    sudo rm -rf /opt/stack/python-proboscis
+    cd /opt/stack
+    git_clone https://github.com/TimSimpsonR/python-proboscis.git \
+        /opt/stack/python-proboscis runs_after_dangerous
+    cd /opt/stack/python-proboscis
+    sudo python setup.py develop
+    cd /tmp
```
Comment (TimSimpsonR) [inline scripts/redstack:236]: Exciting new features.

**Pull request merged**